cluster/gce/configure-vm.sh

```diff
@@ -452,10 +452,11 @@ network_policy_provider: '$(echo "$NETWORK_POLICY_PROVIDER" | sed -e "s/'/''/g")
 enable_manifest_url: '$(echo "${ENABLE_MANIFEST_URL:-}" | sed -e "s/'/''/g")'
 manifest_url: '$(echo "${MANIFEST_URL:-}" | sed -e "s/'/''/g")'
 manifest_url_header: '$(echo "${MANIFEST_URL_HEADER:-}" | sed -e "s/'/''/g")'
-master_name: '$(echo "${MASTER_NAME:-}" | sed -e "s/'/''/g")'
+master_name: '$(echo "${KUBERNETES_MASTER_NAME:-}" | sed -e "s/'/''/g")'
```

Comment:
The use of `master_name` is misguided. The master may not be resolvable even on itself, and this is just not true on AWS:
```
admin@ip-172-20-0-9:~$ ping kubernetes-master
ping: unknown host kubernetes-master
```

I'd like to rip out these pillars completely, and the use in the influxdb PetSet, where it's only used as a uniquifier (instance-prefix is a fine uniquifier).